### PR TITLE
[GIS] All camp locations displayed on Map

### PR DESCRIPTION
**Pull request by imjalpreet**

### CODE REVIEW

File: app/views/camp/view_map.php [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="<?php echo base_url(); ?>static/css/bootstrap.min.css" rel="stylesheet" media="screen">
+    <link href="<?php echo base_url(); ?>static/css/sidebar.css" rel="stylesheet" media="screen">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js"></script>
+    <script src="<?php echo base_url(); ?>static/js/bootstrap.js"></script>
+    <script src="https://maps.googleapis.com/maps/api/js?v=3.exp&sensor=false&libraries=places"></script>
+
+    <script type="text/javascript">
```
Comment (nisargjhaveri) [inline app/views/camp/view_map.php:21]: Shouldn't we make and include a new external JS file?

Comment (imjalpreet) [inline app/views/camp/view_map.php:21]: Ok I will update the PR

**Comment by nisargjhaveri** [comment]
If I have a camp at location A. second one at location B, which is 300 km away from A. Now, when showing all camps on maps, if I select A and 100 km, B still comes up in map by zooming out but there is no camp marker at B.

**Comment by imjalpreet** [comment]
I have updated it a bit, just test it once.

**Comment by nisargjhaveri** [comment]
While showing all camps, can we show the whole world on map?

**Comment by imjalpreet** [comment]
yes i will do that

**Comment by imjalpreet** [comment]
Done!

File: app/views/camp/view_map.php [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+                        </h1>
+                        <br /><br />
+                    </div>
+
+                    <div class="panel panel-primary">
+                        <div class="panel-body">
+                            <label for="my-address"><b>Show Camps Near: </b></label>
+                            <input type="text" class="form-control" placeholder="Enter Location" id="my-address" />
+                            <label for="radius"><b>Search Radius (in Km): </b></label>
+                            <input type="text" class="form-control" placeholder="Enter Search Radius(in Km)" id="radius" />
```
Comment (nisargjhaveri) [inline app/views/camp/view_map.php]: We should also write the default value here, as placeholder maybe.

**Comment by nisargjhaveri** [comment]
Looks good! :+1:

**Pull request merged**